pootle/apps/pootle_store/updater.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.utils import timezone
+from django.utils.functional import cached_property
+
+from pootle_store.constants import POOTLE_WINS
+
+
+class StoreUpdate(object):
```

Comment:
its not really a config object, ive added a docstring with an explanation of its purpose